apps/guardian-ui/src/components/BalanceTable.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  Icon,
+  Box,
+  Flex,
+} from '@chakra-ui/react';
+import { ReactComponent as CheckIcon } from '../assets/svgs/check-circle.svg';
+import { ReactComponent as CloseIcon } from '../assets/svgs/x-circle.svg';
+import { AuditSummary, ModuleKind, MSats } from '@fedimint/types';
+import { formatMsatsToBtc } from '@fedimint/utils';
+
+interface BalanceTableProps {
+  auditSummary: AuditSummary;
+  unit: 'msats' | 'sats' | 'btc';
+}
+
+export const BalanceTable: React.FC<BalanceTableProps> = ({
+  auditSummary,
+  unit,
+}) => {
+  const moduleSummaries = Object.entries(auditSummary.module_summaries).filter(
+    ([, module]) => module.kind !== ModuleKind.Unknown
+  );
```

Comment:
Should we filter the unknown module? It's confusing, but only there in devimint afaik.